modules/microshift-manifests-upgrade-scenario-ostree.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-upgrade-scenario-ostree_{context}"]
+== Manifest upgrade scenario for OSTree systems
+Use the following procedure in the upgrade scenario for OSTree systems where some resources need to be removed while retaining others to preserve data.
+
+[IMPORTANT]
+====
+For OSTree systems, the `delete` subdirectories are read only.
+====
+
+.Procedure
+
+. Identify the manifest that needs updating.
+. Create new manifest for resource deletion that needs to be placed in the `delete` subdirectories.
```

Comment:
Should a new manifest be created ? I think moving the existing manifest to the delete directory ensures that the proper resource is removed ?

Reply:
as per our discussion I am resolving this comment. @dhensel-rh